loader_hub/bilibili/base.py

```diff
@@ -0,0 +1,63 @@
+"""Simple Reader that reads transcript and general infor of Bilibili video."""
+from typing import Any, List, Optional
+import warnings
+
+from gpt_index.readers.base import BaseReader
+from gpt_index.readers.schema.base import Document
+
+
+class BilibiliTranscriptReader(BaseReader):
```

Comment:
You'll need to add this to the `library.json` file for it to be picked up by LlamaIndex's `download_loader` and the LlamaHub website.

Reply:
thanks for the reply! just added.